### DISCUSSION TITLE
Export power meter - exporting more than 5 kW trigger

### DISCUSSION THREAD

**Discussion by cralwalker** [WillCodeForCats/solaredge-modbus-multi#558] [Q&A]
When my system exports more than 5 kW I would like to trigger events. Over the last couple of days I have read the M1 entities and not understood which sensor indicate current power export. I would appreciate if someone could guide me to the right sensor?

**Comment by Uhae** [comment]
I‘m using sensor.se10k_m1_ac_power for such automations. Positive values indicate power being exported.

**Reply by cralwalker** [reply]
Right. I am unclear what the three other sensor measure;
- sensor.se10k_m1_ac_power_a
- sensor.se10k_m1_ac_power_b
- sensor.se10k_m1_ac_power_c
I cannot get them to line up to an equation?

**Reply by Uhae** [reply]
Looks to me like a three-phase ac installation. The sensors you listed measure the power on the three individual phases. By adding them up you should get the total of all phases, which should equal the result of the sensor shown in my previous post.

**Comment by WillCodeForCats** [comment]
What meter type do you have? If it's Export+Import you can use the data directly, if it's Consumption will need a template sensor to calculate it.

**Reply by cralwalker** [reply]
I have the Solaredge SE-MTR-3Y-400V-A export meter.
